src/mantle/events/class-dispatcher.php

```diff
@@ -116,7 +116,11 @@ protected function resolve_subscriber( $subscriber ) {
 	public function dispatch( $event, $payload = [ null ] ) {
 		[ $event, $payload ] = $this->parse_event_and_payload( $event, $payload );
 
-		return apply_filters( $event, ...$payload ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.DynamicHooknameFound
+		if ( function_exists( 'apply_filters' ) ) {
+			return apply_filters( $event, ...$payload ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.DynamicHooknameFound
```

Comment:
If apply filters doesn't exist, do we want to return these values anyway?

Reply:
Mostly a WIP to get my `bin/console` command working.